discovery-provider/src/queries/health_check.py

```diff
@@ -80,7 +81,7 @@ def _get_db_conn_state():
 
 @bp.route("/version", methods=["GET"])
 def version():
-    return jsonify(disc_prov_version), 200
+    return response_with_signature(disc_prov_version)
```

Comment:
since the stuff here in this file doesn't fit the existing format of all json data inside a 'data' property. maybe for now, we can make progress towards unifying that data model? so could we make it something like this? that way in the decode we only take the 'data' and the 'timestamp' explicitly? 
```js
# health_results is some object
response = {**health_results, 'data': health_results}
return response_with_signature(response)
```

Reply:
i am not really in favor of adding duplicate data into the endpoint. it be better to pinpoint the endpoint usages and change the api there. if all the endpoints usages in `health_check.py` are updated, i can go ahead and use the `success_response_with_signature` method (to be renamed to just `success_response) and remove `response_with_signature`

Reply:
so this would only be for /version and /health_check routes. everything else should not be affected by this. sorry if that wasn't clear here. the reason we would need this is to standardize the data format and maintain backwards compatibility with our current libs so our current apps would work

Reply:
in favor of changing the response to contain a `data` key*

Reply:
^ this is exactly it! but i can't think of an easy backwards compatible way of doing this without copying the properties for /health_check and /version